backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -981,7 +981,7 @@ def test_qnn_backend_view_permute_matmul(self):
         sample_input = (torch.randn([1, 8, 512]), torch.randn([1, 2, 8, 256]))
         module = self.get_qdq_module(module, sample_input)
         self.lower_module_and_test_output(module, sample_input)
-        # check if requantization work
+        # check if requantization work by reusing the 8bit qdq module
```

Comment:
@cccclai 
Yes. Please refer to the unit test test_qnn_backend_view_permute_matmul. It demonstrates how mix quantization works.